### PR TITLE
Chore(ci): Introduce GitHub Actions Commitlint workflow

### PR DESCRIPTION
**Pull request by literat**

### CODE REVIEW

File: .github/workflows/commitlint.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+jobs:
+  commitlint:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 5
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 50
+      - uses: actions/setup-node@v1
```
Comment (adamkudrna) [inline .github/workflows/commitlint.yaml:17]: We may want to use newer 2.x version.

https://github.com/actions/setup-node/releases

**Review by adamkudrna — APPROVED** [review]

File: .github/workflows/commitlint.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+---
```
Comment (adamkudrna) [inline .github/workflows/commitlint.yaml:1]: I think this line is not necessary.

File: .github/workflows/commitlint.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+name: Commitlint
+on:
+  push:
+    branches-ignore:
+      - 'main'
```
Comment (adamkudrna) [inline .github/workflows/commitlint.yaml:6]: String is the default data type in YML so i believe the quotes are not needed.

**Review by adamkudrna — CHANGES_REQUESTED** [review]

File: .github/workflows/commitlint.yaml (hunk 1 above)
Comment (literat) [reply .github/workflows/commitlint.yaml]: Ok, updated to v2

File: .github/workflows/commitlint.yaml (hunk 2 above)
Comment (literat) [reply .github/workflows/commitlint.yaml]: Line removed.

File: .github/workflows/commitlint.yaml (hunk 3 above)
Comment (literat) [reply .github/workflows/commitlint.yaml]: Ok, quotes removed.

File: .github/workflows/commitlint.yaml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+          fetch-depth: 50
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 14
+
+      - name: Fetch main
+        run: git fetch --no-tags --depth=50 origin main
+
+      - run: yarn install --frozen-lockfile
+      - name: Commitlint
```
Comment (adamkudrna) [inline .github/workflows/commitlint.yaml:24]: Is there a system of when there is a blank line and when not? 🙂

**Review by adamkudrna — APPROVED** [review]

**Pull request merged**